calc_pi.ipynb

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
<p>Looks like you need a larger <code>N</code> to get a closer estimate of pi?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ojustino/dat_pyinthesky/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Sure, perhaps try 1e5 instead.</p>